packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -39,7 +39,7 @@ export class TreeView extends FASTElement {
             if (this.checkForNestedItems()) {
                 this.slottedTreeItems.forEach(node => {
                     if (isTreeItemElement(node)) {
```

Comment:
I'm noticing `isTreeItemElement` is a type guard for `HTMLElement` - should it be a type guard for `TreeItem`?

Reply:
Technically we support anything with a role of tree item, I'm not sure we want to type guard it for our class here.